src/reducers/url-state.js

```diff
@@ -52,6 +52,17 @@ const dataSource: Reducer<DataSource> = (state = 'none', action) => {
       return state;
   }
 };
+const mouseTimePosition: Reducer<Millseconds | null> = (
```

Comment:
suggestion: It would be good to document this `mouseTimePosition`. I would recommend doing it either here, or on the selector. It should explain what the state means, and how it can be used.

Since this is a partial PR, it's also nice to mention that it's not used yet, and reference the issue number. Something like:

> TODO - This is not used yet, see Issue #222

Cross-referencing issues in code is a great way to let people know what's going on, especially if you don't have time to finish up all the work here.

Reply:
I moved the reducer to profile-view.js, according to Julien's suggestion. Documented it's usage there, also the issue ref.